packages/frontend-api/src/Model/Order/OrderApiFacade.php

```diff
@@ -146,4 +146,24 @@ public function getCustomerOrderCount(Customer $customer): int
     {
         return $this->orderRepository->getCustomerOrderCount($customer);
     }
+
+    /**
+     * @param string $uuid
+     * @param \Shopsys\FrameworkBundle\Model\Customer\Customer $customer
+     * @return \Shopsys\FrameworkBundle\Model\Order\Order
+     */
+    public function getByUuidAndCustomer(string $uuid, Customer $customer): Order
+    {
+        return $this->orderRepository->getByUuidAndCustomer($uuid, $customer);
+    }
+
+    /**
+     * @param string $orderNumber
+     * @param \Shopsys\FrameworkBundle\Model\Customer\Customer $getCustomer
+     * @return \Shopsys\FrameworkBundle\Model\Order\Order
+     */
+    public function getByOrderNumberAndCustomer(string $orderNumber, Customer $getCustomer): Order
```

Comment:
argument should be named just `$customer`, or?

Reply:
fixed: https://github.com/shopsys/shopsys/pull/3296/commits/3483d444204f23c001c587016a6886bef7030342